beacon-chain/rpc/prysm/v1alpha1/validator/proposer_test.go

```diff
@@ -675,11 +677,8 @@ func TestServer_GetBeaconBlock_Electra(t *testing.T) {
 		Graffiti:     graffiti[:],
 	}
 
-	got, err := proposerServer.GetBeaconBlock(ctx, req)
+	_, err = proposerServer.GetBeaconBlock(ctx, req)
```

Comment:
We should get the returned block and compare `got.body.executionRequests` with `blk.body.executionRequests` otherwise we are just not testing much

Reply:
Well, the test was just for the payload, I think this test is useless now.

Reply:
Problem is that we can't pack requests until the engine API changes and we include the requests obtained from the API, I think this falls outside of this PR though